src/js/view/view.js

```diff
@@ -202,6 +202,7 @@ function View(_api, _model) {
         _this.updateBounds();
         _this.updateStyles();
         _this.checkResized();
+        _model.trigger('responsiveUpdate');
```

Comment:
This depends on the player resizing. What if the player does not (when the player size before and after floating is the same)? Shouldn't we use `.addEventListener('transitionend'` to remove `'jw-float-transition'`?

Reply:
The view has the floating controller, so it can invoke a method on it that removes `'jw-float-transition'` rather than triggering an event. We should avoid adding events to the model. We use them to send messages "up" from the floating controller to the view parent, not the other way.